### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Transfers_ScreenReader: When the voiceover tab focus lands on 1st radio button under 'Network Concurrency' it's not read as group begin network concurrency, radio button, 1 of 3, followed by the name of the button. Same issue is seen when shift + Tab to 3rd radio button in the same radio group.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5771]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;

## Environment Details: 
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Settings' pane tree view.
5. Tab to 'Transfers'
6. Tab till 'Adjust Dynamically' radio button under 'Network Concurrency'
7. Turn on voiceover and observe.

## Actual:
When the voiceover tab focus lands on 1st radio button under 'Network Concurrency' it's not read as group begin network concurrency, radio button, 1 of 3, followed by the name of the button. Same issue is seen when shift + Tab to 3rd radio button in the same radio group.

## Expected:
When the voiceover focus lands on 1st and 3rd radio button, the group name network concurrency should be read along with its radio button name and distinct item number.

## User Impact:
Screen reader users will not know which group radio button belongs to.

## Attachment:
![20_NETWORKCONCURRENCY](https://user-images.githubusercontent.com/98393966/169970219-09dd25bb-3c27-444e-9f80-45201c14a6fa.gif)

**Comment by MRayermannMSFT** [comment]
Hi @bbarik06

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-darwin-x64.zip

**Comment by bbarik06** [comment]
Issue is not repro now. Hence, closing the bug. Please refer the attachment.
Version: 1.25.0-dev
<img width="1440" alt="5771_Fixed" src="https://user-images.githubusercontent.com/98393966/176664572-a440d813-2538-491f-a1e4-39d4106aa5e1.png">
